components/mpas-ocean/src/analysis_members/Registry_time_series_stats_monthly_mean.xml

```diff
@@ -199,5 +199,37 @@
                         <var name="velocityMeridionalTimesTemperature_GM"/>
                         <var name="GMBolusVelocityZonal"/>
                         <var name="GMBolusVelocityMeridional"/>
+            <var name="activeTracerForcingMLTend"/>
+            <var name="activeTracerHorAdvectionMLTend"/>
+            <var name="activeTracerHorMixMLTend"/>
```

Comment:
These need to be indented with tabs, not spaces, to be consistent with the other parts of the registry

Reply:
I think the variables also need to be in the same order here as they are in E3SM.  Otherwise, it's really difficult to update them as we go.  Is that already the case?